### PR TITLE
RTED -- second PR

### PR DESCRIPTION
**Pull request by snim2**
Add some getters and setters to the RTED module.

### CODE REVIEW

File: src/lib/rted.rs [hunk 1]
```diff
@@ -177,6 +178,56 @@ impl<'a> RTEDConfig<'a> {
...
+        }
+        if switched {
+            for i in 0..self.size_src {
+                for j in 0..self.size_dst {
+                    if self.itree_src.info[InfoIdx::Post2Size][i]
+                       >= self.itree_dst.info[InfoIdx::Post2Size][j]
+                    {
+                        self.strategy[i][j] = strategy;
+                    } else {
+                        self.strategy[i][j] = strategy + 4;
```
Comment (ltratt) [inline src/lib/rted.rs]: Why `+ 4`? As you know, I really don't like magic constants ;)

Comment (snim2) [reply src/lib/rted.rs]: Hrm. I'm not sure what the `4` means, but it looks like this method isn't called in the original code, so I guess we can rm it for now.

**Comment by snim2** [comment]
Ready for re-review

File: src/lib/rted.rs (hunk 1 above)
Comment (ltratt) [reply src/lib/rted.rs]: I have to ask the obvious question: why did we port a function that isn't used? Does that mean our testing isn't catching this sort of thing?

Comment (snim2) [reply src/lib/rted.rs]: The module is so big I'm splitting it into small PRs. This PR just adds some getters and setters that are not tested, but I had presumed they would be called by methods I haven't implemented yet. However, it looks like this one is just dead code in the Java version.

Comment (ltratt) [reply src/lib/rted.rs]: When do tests get added? I'm not sure we're gaining much if we merge small PRs that don't have tests.

Comment (snim2) [reply src/lib/rted.rs]: It's a good question. The original code has a very small number of tests which can really only be run when the whole algorithm is complete. For these set and get methods I'm not sure there is much that we can usefully do, but I think we can start creating more useful tests from the next PR onwards.

**Comment by ltratt** [comment]
OK, if you rebase, I can merge.

**Pull request merged**